components/organisms/SignUpForm/SignUpFormContent.js

```diff
@@ -111,11 +111,37 @@ const SignUpFormContent = ({ onSubmit }) => {
     submit: values => onSubmit(values),
   };
 
+  const SignInForm = {
+    fields: [
+      {
+        type: 'email',
+        name: 'email',
+        title: 'Email',
+        placeholder: 'Email',
+        initialValue: '',
+        validationSchema: Yup.string()
+          .email('The email must be valid!!')
+          .required('This field is required'),
+      },
+      {
+        type: 'password',
+        name: 'password',
+        title: 'Password',
+        placeholder: '',
+        initialValue: '',
+      },
+      {
+        type: 'submit',
+        name: 'sign_in',
+        initialValue: 'Sign In',
+      },
+    ],
+    submit: values => onSubmit(values),
+  };
   return (
     <>
-      <StyledTitle>Create an account</StyledTitle>
-
-      <Form form={form} />
+      <StyledTitle>{activeForm === 'signUp' ? 'Create an account' : 'Log in'}</StyledTitle>
```

Comment:
Can you compute this values (title, active form) outside from `return` render function? Render function will be a much clearer